sales/send_quotations/terms_and_conditions.rst

```diff
@@ -2,43 +2,98 @@
+invoice that you manage.
 
-Odoo lets you easily include your default terms and conditions on every
-quotation, sales order and invoice.
+Enable this feature
+===================
 
-Set up your default terms and conditions
-========================================
+Go to :menuselection:`Invoicing --> Configuration --> Settings` and activate **Default Terms &
+Conditions** under the **Customer Invoices** category.
 
-Go to :menuselection:`SALES --> Configuration --> Settings` and activate
-*Default Terms & Conditions*.
-
-.. image:: media/terms_and_conditions01.png 
-      :align: center 
+.. image:: media/terms_conditions_1.png
+   :align: center
+   :class: img-thumbnail
+   :alt: How to enable default terms & conditions on Odoo Sales?
  
-In that box you can add your default terms & conditions. They will then
-appear on every quotation, SO and invoice.
+.. tip::
+   Please note that this feature is activated via the settings of the **Invoicing App** and **not**
+   via the settings of the **Sales App**. More than that, you don't need to install the invoicing
+   application because it will automatically be done with the installation of the sales application.
+
+Add terms & conditions to your quotations, sales orders and invoices
+====================================================================
+
+In the settings of the **Invoicing App**, you will have the possibility to insert your default terms
+and conditions.
+
+.. image:: media/terms_conditions_2.png
+   :align: center
+   :class: img-thumbnail
+   :alt: Default terms & conditions on quotation on Odoo Sales?
+
+They will appear subsequently on every quotation, sales order and invoice.
+
+Add terms & conditions to your quotation templates
+==================================================
+
+For even more convenience, you can specify your terms & conditions on every quotation templates
+that you create. These conditions will only appear if you choose to use a distinct quotation
+template when creating a quotation.
 
-.. image:: media/terms_and_conditions02.png 
-  :align: center 
+.. image:: media/terms_conditions_3.png
+   :align: center
+   :class: img-thumbnail
+   :alt: Add terms & conditions to your quotation templates on Odoo Sales?
 
-.. image:: media/terms_and_conditions03.png
-  :align: center
+.. important::
+   Be sure to check out our documentation on how to :doc:`quote_template` to master each step of
+   this amazing feature.
   
-Set up more detailed terms & conditions
-=======================================
+Add more detailed terms & conditions
+====================================
+
+Here are different ideas to share more details about your terms & conditions with your customers:
+
+- Use the **Website App** and create your own terms & conditions page to which you can refer.
+  For example, here is the Odoo terms & conditions page that we send to our customers:
+
+.. image:: media/terms_conditions_4.png
```

Comment:
The whole `image` directive and its sub-directives need to be indented by 2 spaces to be part of the bulleted list.
Also, I recommend to remove the `:align: center` in that case, it renders better in lists ;)